src/controllers/user-statuses-controller.ts

```diff
@@ -21,47 +21,29 @@ export class UserStatusesController extends Controller {
     @Query() userIds: string[],
     @Query() platform: Platform
   ): Promise<UserStatusMap> {
-    // /** Try to retrieve bots from the cache before reading data from the DB. */
-    // const cachedBots: Bots = {};
-    // let failToRetrieveFromCache = false;
-    // /** Iterate on userIds to look for cached data about him. */
-    // for (const userId of userIds) {
-    //   const cachedBot: Bot | string = botsCache.get(`${platform}:${userId}`);
-    //   /** If not *all* users cached, abort and retrieve all from db. */
-    //   if (!cachedBot) {
-    //     failToRetrieveFromCache = true;
-    //     break;
-    //   }
-
-    //   /**
-    //    * If the user marked as 'NOT_EXIST' ignore it.
-    //    */
-    //   if (cachedBot !== 'NOT_EXIST') {
-    //     cachedBots[userId] = cachedBot as Bot;
-    //   }
-    // }
-
-    // /** If all users cached, return the bots from the cache. */
-    // if (!failToRetrieveFromCache) {
-    //   return cachedBots;
-    // }
-
-    /** If cache not hold all user statuses yet, update cache for next time ;) */
-    const userStatuses = await getUserStatusMap(userIds, platform);
-
-    // /** Update cache. */
-    // for (const [userId, confirmedBot] of Object.entries(bots)) {
-    //   botsCache.set(`${platform}:${userId}`, confirmedBot);
-    // }
-
-    // /** Mark all users that not in bots as 'NOT_EXIST' in the cache for next time. */
-    // for (const userId of userIds) {
-    //   if (!(userId in bots)) {
-    //     botsCache.set(`${platform}:${userId}`, 'NOT_EXIST');
-    //   }
-    // }
-
-    return userStatuses;
+    /** Try to retrieve users status from the cache before reading data from the DB. */
+    const usersStatusMap: UserStatusMap = {};
+    for (const userId of userIds) {
+      const cacheResult = await usersCache.get(`${platform}:${userId}`);
```

Comment:
For the next version, we should change that logic to use `mget` and some other structure for the values.
What do you think?